app/src/main/java/com/poupa/vinylmusicplayer/util/PlayingSongDecorationUtil.java

```diff
@@ -78,7 +79,18 @@ public static void decorate(
             @NonNull final AppCompatActivity activity,
             boolean showAlbumImage)
     {
-        final boolean isPlaying = MusicPlayerRemote.isPlaying(song);
+        PlayingSongDecorationUtil.decorate(title, image, imageText, new IndexedSong(song, -1), activity, showAlbumImage);
```

Comment:
Hmm, probably this `-1` `+1` constant should have a name and be used consistently across classes

Reply:
ok